samples/nrf9160/memfault/src/main.c

```diff
@@ -120,12 +120,24 @@ static void button_handler(uint32_t button_states, uint32_t has_changed)
 		LOG_WRN("Stack overflow will now be triggered");
 		fib(10000);
 	} else if (buttons_pressed & DK_BTN2_MSK) {
+		volatile uint32_t i;
+
+#if defined(CONFIG_BUILD_WITH_TFM)
```

Comment:
NITS/Maintainability: Couldn't both TF-M and SPM use "divide by zero"

Reply:
I think the Null Pointer is the more interesting one. And hopefully we can get it working for TF-M as well.

Reply:
Remove the #ifdef